src/components/NavContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+import HomeScreen from '../screens/HomeScreen';
+import TaskScreen from '../screens/TaskScreen';
+import BadgeScreen from '../screens/BadgeScreen';
+import ProfileScreen from '../screens/ProfileScreen';
+
+const Tab = createBottomTabNavigator();
+
+export default function NavFlow() {
+  return (
+    <NavigationContainer>
+      <Tab.Navigator
+        screenOptions={({ route }) => ({
+          tabBarIcon: ({ color, size }) => {
+            let iconName;
+
+            if (route.name === 'Home') {
+              iconName = 'ios-home';
+            } else if (route.name === 'Tasks') {
+              iconName = 'ios-list';
+            } else if (route.name === 'Progress') {
+              iconName = 'ios-clipboard';
+            } else if (route.name === 'Me') {
+              iconName = 'ios-person';
+            }
+
+            return <Ionicons name={iconName} size={size} color={color} />;
+          },
+        })}
+        tabBarOptions={{
+          activeTintColor: '#E96661',
+          inactiveTintColor: 'gray',
+        }}
+      >
```

Comment:
Question for everyone (since I'm not sure myself) - what are people's thoughts on having CSS constants all in a separate file under a JS object? I feel like this would be useful for using the same colors for different components without having to copy/paste but I'm not sure whether this is a frontend convention or not.

E.g. this would become something like `activeTintColor: StyleConstants.ACTIVE_TINT` (and it would be used for other active tints (if any) throughout the app)

Reply:
I personally think this is cleaner and nicer. Not sure what others think about this though

Reply:
@shravanhariharan2 yep, it's the convention. I'd be open to implementing a `src/constants/constants.js` before my next PR. we can store:
- key page colors (aqua green, white, off white)
- font families/size/weights
- context specific constants
- any global timing constants

Reply:
@unsoupervised This is a great idea, please go ahead with that change.